src/renderer/components/opds/BrowserResult.tsx

```diff
@@ -140,6 +151,10 @@ export class BrowserResult extends React.Component<BrowserResultProps, null> {
             return newUrl;
         }
     }
+
+    private goto(url?: string) {
```

Comment:
I don't think `url` should be an optional (i.e. possibly `undefined`) function argument. In fact, callers of the `goto()` function ensure it is never called with a falsy value.